docs/02-app/01-building-your-application/01-routing/05-dynamic-routes.mdx

```diff
@@ -21,9 +21,15 @@ Dynamic Segments are passed as the `params` prop to [`layout`](/docs/app/api-ref
 
 For example, a blog could include the following route `app/blog/[slug]/page.js` where `[slug]` is the Dynamic Segment for blog posts.
 
-```tsx filename="app/blog/[slug]/page.js"
+```tsx filename="app/blog/[slug]/page.tsx" switcher
+export default function Page({ params: { slug: string } }) {
+  return <div>My Post: {slug}</div>
+}
+```
+
+```jsx filename="app/blog/[slug]/page.js" switcher
```

Comment:
```suggestion
```jsx filename="app/blog/[slug]/page.jsx" switcher
```

Reply:
According to @delbaoliveira [chart](https://github.com/vercel/next.js/pull/51056#issuecomment-1586036686), no.